.Lib9c.Tests/Model/RankingSimulatorTest.cs

```diff
@@ -167,5 +167,71 @@ public void ConstructorWithCostume()
             var enemyPlayer = (EnemyPlayer)e.Character;
             Assert.Equal(row2.Stat, enemyPlayer.Stats.OptionalStats.DEF);
         }
+
+        [Theory]
+        [InlineData(1, 1000)]
+        [InlineData(55, 1000)]
+        [InlineData(85, 1000)]
+        [InlineData(120, 1000)]
+        [InlineData(160, 1000)]
+        [InlineData(200, 1000)]
+        public void CheckToReceiveAllRewardItems(int level, int simulationCount)
+        {
+            _tableSheets.WeeklyArenaRewardSheet = _tableSheets.WeeklyArenaRewardSheet;
+            var avatarState = new AvatarState(
+                default,
+                default,
+                0,
+                _tableSheets.GetAvatarSheets(),
+                new GameConfigState(),
+                default)
+            {
+                level = level,
+            };
+            avatarState.worldInformation.ClearStage(
+                1,
+                GameConfig.RequireClearedStageLevel.ActionsInRankingBoard,
+                1,
+                _tableSheets.WorldSheet,
+                _tableSheets.WorldUnlockSheet);
```

Comment:
```suggestion
            avatarState.worldInformation = new WorldInformation(
                0,
                _tableSheets.WorldSheet,
                GameConfig.RequireClearedStageLevel.ActionsInRankingBoard);
```

`WorldInformation.ClearStage()` works only when the previous stage of the stage received as an argument is cleared. So it's easy to write it like this.